datasets/us/trade_csl/us_trade_csl.yml

```diff
@@ -100,6 +114,277 @@ lookups:
         properties:
           name: Ministry of Defence of the Russian Federation
           notes: Includes the Armed Forces of Russia and all operating units wherever located.  This includes the national armed services (army, navy, marine, air force, or coast guard), as well as the national guard and national police, government intelligence or reconnaissance organizations of the Russian Federation.  All address located in Russia.
+      - match: 'Huawei Technologies Co., Ltd. (Huawei).  Affiliated Entity:  Beijing Huawei Longshine Information Technology Co., Ltd.'
+        properties:
+          name: Huawei Technologies Co., Ltd. (Huawei)
+          related: Beijing Huawei Longshine Information Technology Co., Ltd
+          notes: 'Affiliated Entity:  Beijing Huawei Longshine Information Technology Co., Ltd.'
+      - match: "Huawei Technologies Co., Ltd. (Huawei).  Affiliated Entity: Hangzhou New Longshine Information Technology Co., Ltd."
+        properties:
+          name: Huawei Technologies Co., Ltd. (Huawei)
+          related: Hangzhou New Longshine Information Technology Co., Ltd
+          notes: "Affiliated Entity: Hangzhou New Longshine Information Technology Co., Ltd."
+      - match: "Huawei Technologies Co., Ltd. (Huawei).  Affiliated Entity:  Hangzhou Huawei Communication Technology Co., Ltd."
+        properties:
+          name: Huawei Technologies Co., Ltd. (Huawei)
+          related: Hangzhou Huawei Communication Technology Co., Ltd
+          notes: "Affiliated Entity:  Hangzhou Huawei Communication Technology Co., Ltd."
+      - match: "Huawei Technologies Co., Ltd. (Huawei).  Affiliated Entity:  Huawei Digital Technologies (Suzhou) Co., Ltd."
+        properties:
+          name: Huawei Technologies Co., Ltd. (Huawei)
+          related: Huawei Digital Technologies (Suzhou) Co., Ltd
+          notes: "Affiliated Entity:  Huawei Digital Technologies (Suzhou) Co., Ltd."
+      - match: "Huawei Technologies Co., Ltd. (Huawei).  Affiliated Entity:  Hangzhou Huawei Enterprises"
+        properties:
+          name: Huawei Technologies Co., Ltd. (Huawei)
```

Comment:
I wonder whether it should actually be the other way round: "Hangzhou Huawei Enterprises" is the entity being targeted by this entry, and they are related to "Huawei Technologies Co., Ltd. (Huawei)". What do you think?